actions/workflows/create_vm_windows.yaml

```diff
@@ -194,7 +200,19 @@ tasks:
       - when: <% succeeded() %>
         publish:
           - cname: <% ctx().hostname %>.<% ctx().dns_zone %>
+      - when: <% failed() %>
+        do:
+          - destroy_vm_on_failure
+
   destroy_vm_on_failure:
+    next:
```

Comment:
Shouldn't we run st2cd.destroy_vm here and then when action complete explicitly fail the workflow?

Reply:
Shouldn't destroy_vm_on_failure looks like https://github.com/StackStorm/st2cd/blob/d5184d26e44aa437b33446f4e3478ed41429cac5/actions/workflows/create_vm_role.yaml#L141?